src/output-json-alert.c

```diff
@@ -174,6 +174,53 @@ static void AlertJsonDnp3(const Flow *f, json_t *js)
     return;
 }
 
+static void AlertJsonSourceTarget(const Packet *p, const PacketAlert *pa, json_t *js, json_t* ajs)
+{
+        json_t *sjs = json_object();
+        if (sjs == NULL) {
+            json_decref(js);
```

Comment:
It was also done in AlertJsonHeader so I did mimic. I'm changing that and fixing it in other places too.